js/framework/AnalyticsConstants.js

```diff
@@ -446,8 +449,76 @@ if (!OO.Analytics.EVENT_DATA)
   EVENT_DATA.AdPodEndedData = function(adId)
   {
     var checkAdPodEndedData = OO._.bind(checkDataType, this, "AdPodEndedData");
-    this.adId = checkAdPodEndedData(adId, "adId", "string");
-  };
+    this.adId = checkAdPodEndedData(adId, "adId", ["string"]);
+  }
+
+  /**
+   * @public
+   * @class Analytics.EVENT_DATA#AdStartedData
+   * @classdesc Contains information about the type of ad that has started and its ad data.
+   * @property {string} adType The type of ad (linear video, linear overlay, nonlinear overlay)
+   * @property {object} adMetadataIn The metadata associated with the ad
+   */
+  EVENT_DATA.AdStartedData = function(adType, adMetadataIn)
+  {
+    var checkAdStartedData = OO._.bind(checkDataType, this, "AdStartedData");
+    this.adType = checkAdStartedData(adType, "adType", ["string"]);
+    this.adMetadata = checkAdType(adType, adMetadataIn);
```

Comment:
The data is different from what it was. adId, adDuration, adPodPosition is now stored in adMetadata for linear ads. Will we need to update the plugins to match with this update? Nielsen also wants us to ignore nonlinear overlays for their reporting. Will we need to make a change in the plugins for this as well?